Source/Optional.swift

```diff
@@ -84,3 +84,5 @@ public extension Optional {
         return f.flatMap { self.map($0) }
     }
 }
+
+let x = "actually break it" as! String
```

Comment:
Name Format Violation: Variable name should be between 3 and 40 characters in length: 'x'